test/cpp_extensions/libtorch_agnostic_extension/libtorch_agnostic/csrc/kernel.cpp

```diff
@@ -478,6 +478,62 @@ STABLE_TORCH_LIBRARY_IMPL(libtorch_agnostic, CompositeExplicitAutograd, m) {
   m.impl("my_amax_vec", &boxed_my_amax_vec);
 }
 
+std::vector<Tensor> my__foreach_mul(torch::headeronly::HeaderOnlyArrayRef<Tensor> self, torch::headeronly::HeaderOnlyArrayRef<Tensor> other) {
+  const auto num_args = 2;
+  StableIValue stack[num_args];
+  stack[0] = from(self);
+  stack[1] = from(other);
```

Comment:
nit: this is acceptable, but since num_args isn't actually used anywhere else and I hate being nagged by linters, I would write

std::array<StableIValue, 2> stack = {{from(self), from(other)}};